src/coreclr/vm/cgensys.h

```diff
@@ -97,8 +97,14 @@ inline void GetSpecificCpuInfo(CORINFO_CPU * cpuInfo)
 // MSVC directly defines intrinsics for __cpuid and __cpuidex matching the below signatures
 // We define matching signatures for use on Unix platforms.
 
-extern "C" void __stdcall __cpuid(int cpuInfo[4], int function_id);
-extern "C" void __stdcall __cpuidex(int cpuInfo[4], int function_id, int subFunction_id);
+#ifdef TARGET_X86
+#define CALLCONV __cdecl
+#else
+#define CALLCONV __stdcall
+#endif
+
+extern "C" void CALLCONV __cpuid(int cpuInfo[4], int function_id);
+extern "C" void CALLCONV __cpuidex(int cpuInfo[4], int function_id, int subFunction_id);
 #endif // TARGET_UNIX
 extern "C" DWORD __stdcall xmmYmmStateSupport();
```

Comment:
You can delete explicit `__stdcall` here as well while you are on it. (It needs to be deleted in the implementation as well.)